joinmarket-webui/joinmarket-webui
# ui: Add content placeholders

This PR replaces loaders with content placeholders. By doing this we remove jitter and give the user a visual hint. This is especially useful in the `Send` form, where the user can see what information needs to be entered.

I've opted to use the `Placeholder` component from `react-bootstrap` instead of introducing an additional lib.

**Before:**
https://www.loom.com/share/fc39755d2a1f4f288cb7419470cf08ba

**After:**
https://www.loom.com/share/ac93b1ac92994390a51ef49a5f490b41

**How to test:**
Run locally, make sure all buttons are disabled and that no action can be taken before loading is complete. 

I'm fairly new to the codebase/app so please test carefully. All suggestions are welcome!
